models/movie.go

```diff
@@ -0,0 +1,58 @@
+package models
+
+import (
```

Comment:
As a best practice, the imports should be separated as follows:

```go
import(
    Go core
    
    Project local
    
    Third party
)
```

Reply:
Updated!